cmd/dep/main.go

```diff
@@ -55,13 +55,44 @@ func main() {
 		fmt.Fprintln(os.Stderr)
 	}
 
-	if len(os.Args) <= 1 || len(os.Args) == 2 && (strings.Contains(strings.ToLower(os.Args[1]), "help") || strings.ToLower(os.Args[1]) == "-h") {
+	// parseArgs determines the name of the dep command and wether the user asked for
+	// help to be printed.
+	parseArgs := func(args []string) (cmdName string, printCmdUsage bool, exit bool) {
```

Comment:
It's large enough and has it's own docs, so it should be pulled up IMO. If anyone else thinks I'm wrong we can certainly change that in another PR though.